src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1001,7 +1001,7 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: DataSource,
             submissionEntity = Option(AttributeEntityReference(entityType = submissionRequest.entityType, entityName = submissionRequest.entityName)),
             workflows = succeededWorkflowSubmissions,
             notstarted = failedWorkflows,
-            status = if (submittedWorkflows.isEmpty) SubmissionStatuses.Done else SubmissionStatuses.Submitted
+            status = if (succeededWorkflowSubmissions.isEmpty) SubmissionStatuses.Done else SubmissionStatuses.Submitted
```

Comment:
submittedWorkflows also contained workflows that ended up failing due to things like 504 - so we needed to instead only check succeededWorkflowSubmissions here, which are workflows things that actually got handled by Cromwell